crowbar_framework/config/routes.rb

```diff
@@ -75,6 +75,13 @@
   post "utils/repositories/deactivate", controller: "repositories", action: "deactivate", as: "deactivate_repository"
   post "utils/repositories/activate_all", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
   post "utils/repositories/deactivate_all", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
+  get "utils/backup", controller: "backup", action: "index", as: "backup"
+  get "utils/backup/download/:name/:created_at", controller: "backup", action: "download", as: "backup_download"
```

Comment:
Metrics/LineLength: Line is too long. [112/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)